### PR TITLE
Cloning Apple Solo Project

### PR DESCRIPTION
**Pull request by Ferm0494**
Made Html Structure, CSS and Linter and Error Fixing.

### CODE REVIEW

**Review by serdg0 — CHANGES_REQUESTED** [review]
**Status: Required Changes.**

Hello @Ferm0494 ! :v: 

You made a great work in this one! But there are some things you could do to make this project **outstanding**! :1st_place_medal: 

This project is about playing and understanding background images and gradients.
Here they are:

**Readme**

- [x] Readme Doesn't have author name

**Body**

- [x] The body of the pŕoject have a margin. Let's get rid of it.

**Main cover**
For this section I attach the image comparison.
**Yours**
Selection_013
**Original**
Selection_014

- [x] Allign the background image, and the texts, so it fits like the original one.

**Navbar**

- [x] Each `li` element should be a link (#).

- [x] Add the support link.

**4-article section**

- [x] Each articile should be a link (#)

Kindly make those changes and submit for another code review.

You've done a great job so far! Keep the good work! :smiley: :+1: 

If anything, feel free to reach me in slack.

And never forget to...

### Have fun coding! :computer: :computer: 

Sergio Diaz Gonzalez.

**Comment by sumancrest0001** [comment]
### STATUS: REQUIRED CHANGES
Hello @Ferm0494, You have fixed issues stated by the previous TSE. :clap:  :clap: 
But It looks like you forgot to add links to each of the articles just below the main section. We need to fix few other of issues to make it similar to that of the original.

File: index.html
Comment (sumancrest0001) [inline index.html]: [REQUIRED] Please add <a> tag on each image.

**Comment by sumancrest0001** [comment]
[REQUIRED] Please try to emulate the footer section similar to that of the original one. Add margins at both sides, reduce height of vertical lines.

currently :point_down: 
Screenshot from 2020-04-11 12-31-44

should be like this :point_down:  :point_down: 
Screenshot from 2020-04-11 12-32-00

**Comment by sumancrest0001** [comment]
[Optional] Add underline when hovered over this section.
Screenshot from 2020-04-11 12-32-26

**[NOTE] It is not compulsory to fix the optional change.**

You are an inch closer to get approval for this project. Please make the above changes and submit the project for another review.

**Comment by sumancrest0001** [comment]
**Need help? Do not hesitate to contact me on slack **Suman Shrestha****

**Review by men32z — APPROVED** [review]
## Good job making the changes, you can merge.
before to merge consider the following changes
- [ ] as the previous TSE recommended add underline in the text mentioned.
- [ ] in the original page all the content has a limit of 1440px and the content is centered, try to implement the same so your project will look better.
- [ ] your gradient and navbar are great and meet the requirements but still looking different as the original project. try to make it look as original.
- [ ] the main image shouldn't be distorted in different sizes.

#### Required in the next projects
- [ ] add a descriptive Pull Request, you should describe the features of the project.

BlgS1sdIUAEybkW

### Status Approved

--Luis Preza

**Pull request merged**
